tests/unit/controllers/signup-test.js

```diff
@@ -5,23 +5,7 @@ module('Unit | Controller | signup', function (hooks) {
   setupTest(hooks);
   test('it test', function (assert) {
     let controller = this.owner.lookup('controller:signup');
-    assert.expect(46);
-    assert.equal(
-      controller.isSubmitClicked,
-      false,
-      'initially isSubmitClicked is false'
-    );
-    assert.equal(
-      controller.isButtonDisabled,
-      true,
-      'initially isButtonDisabled is true'
-    );
-    assert.equal(
-      controller.state,
-      'get-started',
-      'initially state is get-started'
-    );
-    assert.equal(controller.dev, false, 'initially dev is false');
+    assert.expect(42);
```

Comment:
What is the significance of this value ??
Can we please make a constant to understand it better ?

Reply:
assert.expect is the number of assert we will have in that particular test. Does make sense to make it a constant I guess